src/Components/Event/Event.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect} from 'react'
+import EventForm from '../EventForm/EventForm'
+//
+const Event = () => {
```

Comment:
We'll have to get this mutation hooked up & then we're all set. Awesome job here!